apps/arweave/src/ar.erl

```diff
@@ -586,6 +601,26 @@ start(Config) ->
 	end,
 	start_dependencies().
 
+validate_cm_pool_config(Config) ->
+	case {Config#config.coordinated_mining, Config#config.pool_server} of
+		{true, true} ->
+			io:format("~nThe pool server node cannot participate "
```

Comment:
Possible to add a comment explaining why the server can't participate in coordinated mining?

Reply:
I was defensive here blocking the use-case which I find unlikely to be useful. However, I cannot see why it won't work at the moment so we can enable it if you prefer.

Reply:
Got it. Nope I like your approach of preventing unlikely/unuseful use cases to limit complexity! My comment was more because I didn't understand why it cannot participate technically - but since that's not the case, no need to elaborate further. Your code as is works for me!